### PR TITLE
[VEN-2515] Sweep Tokens in XVSVaultTreasury

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN-2515

### CODE REVIEW

File: tests/ProtocolReserve/XVSVaultTreasury.ts [hunk 1]
```diff
@@ -112,4 +112,47 @@ describe("XVS vault treasury: tests", () => {
       await expect(tx).to.emit(xvsVaultTreasury, "FundsTransferredToXVSStore");
     });
   });
+
+  describe.only("Sweep tokens", () => {
```
Comment (chechu) [inline tests/ProtocolReserve/XVSVaultTreasury.ts:116]:
```suggestion
  describe("Sweep tokens", () => {
```

File: contracts/ProtocolReserve/XVSVaultTreasury.sol [hunk 2]
```diff
@@ -83,6 +86,29 @@ contract XVSVaultTreasury is AccessControlledV8, ReentrancyGuardUpgradeable {
         emit FundsTransferredToXVSStore(xvsStore, amountMantissa);
     }
 
+    /// @notice Function to sweep tokens from the contract
+    /// @param tokenAddress Address of the asset(token)
+    /// @param to Address to which assets will be transferred
+    /// @param amount Amount need to sweep for the contract
```
Comment (Debugger022) [inline contracts/ProtocolReserve/XVSVaultTreasury.sol:92]:
```suggestion
    /// @param amount Amount need to sweep from the contract
```

Comment (Debugger022) [reply contracts/ProtocolReserve/XVSVaultTreasury.sol:92]: I see that you have deployed the new implementation with this new function, So, you can also ignore this change.

**Comment by Debugger022** [comment]
LGTM

**Review by Debugger022 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**